NoSQL/README.md

```diff
@@ -98,4 +148,4 @@ Oracle NoSQL Community Edition has **no** commercial support.
 
 ## Copyright
 
-Copyright (c) 2017, 2020 Oracle and/or its affiliates.
+Copyright (c) 2017, 2022 Oracle and/or its affiliates.
```

Comment:
Please replace/update the copyright header on all modified files, not just this one.